test_filter.py

```diff
@@ -174,14 +175,24 @@ def test_contains_regex_parse_error(self):
         params = simple_params("A", "text_contains_regex", "*", case_sensitive=True)
         result = render(table, params)
         self.assertEqual(
-            result, "Regex parse error: no argument for repetition operator: *"
+            result, 
+            i18n_message(
+                "RegexParseError.message",
```

Comment:
Please make these message IDs start lowercase. It looks strange for them to start uppercase.